src/styles/components/quota/styles.less

```diff
@@ -26,3 +30,13 @@
     }
   }
 }
+
+.quota-info {
```

Comment:
Please don't override ui-kit styles. The designers understand that we will always use ui-kit styles, even if their mocks are slightly different